stdlib/public/Darwin/Accelerate/vImage_CVImageFormat.swift

```diff
@@ -134,12 +134,22 @@ extension vImageCVImageFormat {
         #if os(OSX)
         return String(UTCreateStringForOSType(formatCode).takeRetainedValue())
         #else
-        let chars: [UInt8] = [UInt8(formatCode >> 24 & 0xFF),
-                              UInt8(formatCode >> 16 & 0xFF),
-                              UInt8(formatCode >> 8 & 0xFF),
-                              UInt8(formatCode >> 0 & 0xFF)]
+        let chars: [UnicodeScalar?] = [UnicodeScalar(formatCode >> 24 & 0xFF),
+                                       UnicodeScalar(formatCode >> 16 & 0xFF),
+                                       UnicodeScalar(formatCode >> 8 & 0xFF),
+                                       UnicodeScalar(formatCode >> 0 & 0xFF)]
+        
+        var returnString: String = ""
+        
+        chars.compactMap {
+            $0
+            }.compactMap{
+                String($0)
+            }.forEach {
+                returnString.append($0)
+        }
```

Comment:
What's the benefit of exposing this property as a `String`? How is a developer going to use this in practice, and is String really preferred to UInt32 for that purpose?

Reply:
Good point, I've changed back to simply returning the `UInt32`.